src/config/frontend/EdFi.DmsConfigurationService.Frontend.AspNetCore/Model/ApplicationValidator.cs

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DmsConfigurationService.DataModel;
+using FluentValidation;
+
+namespace EdFi.DmsConfigurationService.Frontend.AspNetCore.Model;
+public class ApplicationValidator : AbstractValidator<Application>
+{
+    public ApplicationValidator()
+    {
+        RuleFor(a => a.ApplicationName).NotEmpty().MaximumLength(256);
+        RuleFor(a => a.VendorId)
```

Comment:
You don't need any of these validators but you probably do need something to make sure the vendorId is a valid Vendor.Id in the database. We can handle it at the database level by catching that error and returning a new type of result `FailureReferenceNotExists` or something. See my other comment in the integration test.